CONTRIBUTING.md

```diff
@@ -1,53 +1,53 @@
 # Contributing
 
-The release of Bootstrap 4 and Rails 5.1 have implications
-for the future direction of `bootstrap_form`.
-Don't worry.
-We plan to move this gem forward to Bootstrap 4
-and to support Rails 5.1 and beyond.
-If you're thinking of contributing to `bootstrap_form`,
-please read [issue #361](https://github.com/bootstrap-ruby/rails-bootstrap-forms/issues/361).
-Your comments are welcome.
-
-Thanks so much for considering contributing to bootstrap_form!
-We love pull requests!
+The release of Bootstrap 4 and Rails 5.1 have implications for the future
+direction of `bootstrap_form`. Don't worry. We plan to move this gem forward to
+Bootstrap 4 and to support Rails 5.1 and beyond. If you're thinking of
+contributing to `bootstrap_form`, please read
+[issue #361](https://github.com/bootstrap-ruby/rails-bootstrap-forms/issues/361).
```

Comment:
I'd like to keep the friendly and welcoming attitude (perhaps with some grammar edits):

```
Your comments are welcome.

Thanks so much for considering a contribution to bootstrap_form.
We love pull requests!
```

Any reason to remove it?

Reply:
I think we want to make the CONTRIBUTING document more complete, rather than less.

Reply:
Felt like it was kinda redundant. I'll add it back.